app/src/main/java/com/hedvig/app/feature/home/ui/claimstatus/data/RelatedContractTypeData.kt

```diff
@@ -0,0 +1,16 @@
+package com.hedvig.app.feature.home.ui.claimstatus.data
+
+import androidx.compose.runtime.Composable
+import com.hedvig.android.owldroid.graphql.HomeQuery
+
+@JvmInline
+value class RelatedContractTypeData(val text: String) {
+    companion object {
+        @Composable
+        fun fromClaimStatus(claimStatus: HomeQuery.ClaimStatus): RelatedContractTypeData {
+            // TODO Is displayName appropriate?
+            val relatedContract = claimStatus.contract?.displayName ?: String()
```

Comment:
```suggestion
            val relatedContract = claimStatus.contract?.displayName ?: ""
```

Or does `String()` have other benefits? 😄